cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -50,6 +50,15 @@ def test_infra_mapping_ui_assertions(appliance, v2v_providers, form_data_single_
      mapping_list.get_map_source_networks(mapping.name)[0])
     soft_assert(mapping.form_data['network'].values()[0]['mappings'][0]['target'][0].format() in
      mapping_list.get_map_target_networks(mapping.name)[0])
+
+    rhv_prov = v2v_providers.rhv_provider
+    host = rhv_prov.hosts.all()[0]
+    host.refresh(cancel=True)
+    view = navigate_to(infrastructure_mapping_collection, 'All', wait_for_view=True)
+    mapping_list = view.infra_mapping_list
+    soft_assert(mapping_list.get_map_source_networks(mapping.name) != [])
```

Comment:
Optional:
@kedark3: when we compare with the empty list. use of `not` is more pythonic than it like...

```
In [8]: y = []

In [9]: x
Out[9]: [1, 2, 3]

In [10]: not x
Out[10]: False

In [11]: not y
Out[11]: True

In [12]: not x
Out[12]: False
```

Reply:
`not mapping_list.get_map_source_networks(mapping.name)` yields `false` as the list is not empty, and assertion fails, as assertion wants `true` so instead of saying `not mapping_list.get_map_source_networks(mapping.name) == False` I chose to use original `!= []`